src/Aeson.purs

```diff
@@ -344,6 +345,13 @@ decodeJsonString = parseJsonStringToAeson >=> decodeAeson
 
 -------- DecodeAeson instances --------
 
+instance DecodeAeson UInt where
```

Comment:
I wonder if there's any way we can factor out some of these instances for integral types, like `(String -> Maybe a) -> Aeson -> Either JsonDecodeError a`